### ISSUE TITLE
There are several CVEs in the project dependencies

### ISSUE THREAD

**Issue by Security3rd** [eclipse/jetty.project#4616]
I found your project used some dependencies that contain CVEs. I suggest a library update to avoid potential risks. Details are listed below:
* **Vulnerable Library Version:** org.apache.derby : derby : 10.4.1.3
  **CVE ID:** [CVE-2009-4269](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2009-4269), [CVE-2015-1832](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2015-1832), [CVE-2010-2232](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2010-2232)
  **Import Path:** tests/test-loginservice/pom.xml
  **Suggested Safe Versions:** 10.12.1.1, 10.13.1.1, 10.14.1.0, 10.14.2.0, 10.15.1.3
* **Vulnerable Library Version:** org.apache.derby : derby : 10.1.2.1
  **CVE ID:** [CVE-2006-7217](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2006-7217), [CVE-2006-7216](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2006-7216), [CVE-2015-1832](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2015-1832), [CVE-2009-4269](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2009-4269), [CVE-2010-2232](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2010-2232)
  **Import Path:** jetty-plus/pom.xml
  **Suggested Safe Versions:** 10.12.1.1, 10.13.1.1, 10.14.1.0, 10.14.2.0, 10.15.1.3
* **Vulnerable Library Version:** com.hazelcast : hazelcast : 3.9.4
  **CVE ID:** [CVE-2016-10750](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2016-10750)
  **Import Path:** jetty-hazelcast/pom.xml, jetty-hazelcast/pom.xml
  **Suggested Safe Versions:** 3.10.7, 3.11, 3.11-BETA-1, 3.11.1, 3.11.2, 3.11.3, 3.11.4, 3.11.5, 3.11.6, 3.11.7, 3.12, 3.12-BETA-1, 3.12-BETA-2, 3.12.1, 3.12.2, 3.12.3, 3.12.4, 3.12.5, 3.12.6, 4.0, 4.0-BETA-1, 4.0-BETA-2
* **Vulnerable Library Version:** org.glassfish.web : javax.servlet.jsp.jstl : 1.2.2
  **CVE ID:** [CVE-2015-0254](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2015-0254)
  **Import Path:** jetty-osgi/test-jetty-osgi/pom.xml
  **Suggested Safe Versions:** 1.2.3, 1.2.4, 1.2.5, 1.2.5-b01, 1.2.5-b02, 1.2.5-b03

**Comment by olamy** [comment]
Good catch regarding Hazelcast but except Hazelcast everythinh else is used for test.
Maybe you should probably update your robot to not report error regarding dependencies with scope test?
